### PR TITLE
feat: implement initial version of  cmd-nse-vl3-vpp

### PR DESCRIPTION
**Pull request by denis-tingaikin**
## Motivation

Closes https://github.com/networkservicemesh/cmd-nse-vl3-vpp/issues/1
Closes https://github.com/networkservicemesh/cmd-nse-vl3-vpp/issues/2
Closes https://github.com/networkservicemesh/cmd-nse-vl3-vpp/pull/3

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -0,0 +1,490 @@
...
+	"github.com/networkservicemesh/sdk/pkg/tools/debug"
+	"github.com/networkservicemesh/sdk/pkg/tools/grpcutils"
+	"github.com/networkservicemesh/sdk/pkg/tools/log"
+	"github.com/networkservicemesh/sdk/pkg/tools/log/logruslogger"
+	"github.com/networkservicemesh/sdk/pkg/tools/spiffejwt"
+)
+
+// Config holds configuration parameters from environment variables
+type Config struct {
+	Name                  string            `default:"icmp-server" desc:"Name of ICMP Server"`
```
Comment (edwarnicke) [inline main.go:82]: Why is this called icmp-server?

File: main.go [hunk 2]
```diff
@@ -0,0 +1,490 @@
...
+	DialTimeout           time.Duration     `default:"5s" desc:"timeout to dial NSMgr" split_words:"true"`
+	RequestTimeout        time.Duration     `default:"15s" desc:"timeout to request NSE" split_words:"true"`
+	ListenOn              string            `default:"listen.on.sock" desc:"listen on socket" split_words:"true"`
+	ConnectTo             url.URL           `default:"unix:///var/lib/networkservicemesh/nsm.io.sock" desc:"url to connect to" split_words:"true"`
+	MaxTokenLifetime      time.Duration     `default:"10m" desc:"maximum lifetime of tokens" split_words:"true"`
+	ServiceNames          []string          `default:"icmp-responder" desc:"Name of providing service" split_words:"true"`
```
Comment (edwarnicke) [inline main.go:88]: Why does this default to I map server?

File: main.go [hunk 3]
```diff
@@ -0,0 +1,490 @@
...
+	Payload               string            `default:"IP" desc:"Name of provided service payload" split_words:"true"`
```
Comment (edwarnicke) [inline main.go:89]: This should always be IP for vL3

File: main.go (hunk 1 above)
Comment (denis-tingaikin) [reply main.go]: Thanks, fixed!

File: main.go (hunk 2 above)
Comment (denis-tingaikin) [reply main.go]: Thanks, fixed!

File: main.go (hunk 3 above)
Comment (denis-tingaikin) [reply main.go]: Thanks, fixed!

**Pull request merged**